tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -61,10 +65,13 @@ describe(__filename, () => {
   it('does not dispatch fetchUserCollections if there is no user', () => {
     const { store } = dispatchClientMetadata();
     const fakeDispatch = sinon.spy(store, 'dispatch');
+    const errorHandler = createStubErrorHandler();
 
-    renderComponent({ store });
+    renderComponent({ store, errorHandler });
```

Comment:
There is no need to pass an `errorHandler` into `renderComponent` here, because it doesn't affect the outcome of the test and we never assert anything about it.